src/style.css

```
@@ -0,0 +1,44 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars_star {
+  background-image: url(images/star.svg);
+  height: 16px;
+  width: 16px;
+  border-radius: 0.5px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars_star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--5 > .stars_star {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--4 > .stars_star:nth-of-type(-n + 4) {
+  background-image: url(images/star-active.svg);
+}
```

```suggestion
.stars--5 > .stars_star,
.stars--4 > .stars_star:nth-of-type(-n + 4) {
  background-image: url(images/star-active.svg);
}
```

use commas to combine selectors